lib/ramble/ramble/language/shared_language.py

```diff
@@ -201,7 +201,7 @@ def _execute_success_criteria(obj):
             'file': file,
             'fom_name': fom_name,
             'fom_context': fom_context,
-            'formula': formula,
+            'formula': formula
```

Comment:
Is there a reason the comma had to be removed?

Reply:
No, it just made it more consistent with the rest of the file. Happy to put it back / make all in the file have trailing `,` / leave as is